protobuf/lib/src/protobuf/mixins/well_known.dart

```diff
@@ -537,7 +539,7 @@ mixin ListValueMixin implements GeneratedMessage {
     if (json is List) {
       final subBuilder = message.info_.subBuilder(_valueFieldTagNumber)!;
       for (var i = 0; i < json.length; i++) {
-        final Object element = json[i];
+        final Object? element = json[i];
```

Comment:
Did you run `dart fix` or similar? Can we avoid code changes in formatting focused PR?